package.json

```diff
@@ -8,7 +8,9 @@
     "contentful": "^4.3.0",
     "contentful-management": "^3.4.0",
     "nuxt": "^1.0.0-rc4",
-    "vue-markdown": "^2.1.3"
+    "vue-marked": "^0.1.1",
+    "vue-server-renderer": "2.5.11",
+    "vue-template-compiler": "2.5.11"
```

Comment:
Maybe I overlook something but are these two (`vue-server-renderer` and `vue-template-compiler`) used right now?

Reply:
Oops. Not sure how those ended up in there. 😊